doc/tutorials/index.md

```diff
@@ -0,0 +1,38 @@
+# Tutorials
+
+Welcome to the Tutorials section. Here, you will find clear, step-by-step guides to help you master hvPlot and explore the full holoviz ecosystem. Whether you’re just starting out, coming from pandas or xarray, or looking to sharpen your skills, these tutorials offer practical examples and hands-on learning to explore and transform your data into compelling visualizations.
+
+<div class="card-container">
```

Comment:
Why not reuse what Panel did? For instance with https://github.com/holoviz/panel/blob/main/doc/how_to/index.md.

Reply:
OK

Reply:
Looks better now IMO

![image](https://github.com/user-attachments/assets/43e59541-d949-48d3-9749-e438f0eafa0f)